pkg/pgmodel/migrations/sql/idempotent/base.sql

```diff
@@ -1837,13 +1889,15 @@ CALL execute_everywhere($ee$
                 PERFORM decompress_chunk(format('%I.%I', chunk_row.schema_name, chunk_row.table_name)::regclass);
             END IF;
 
-            COMMIT;
+            IF NOT transactional THEN
+              COMMIT;
+            END IF;
         END LOOP;
     END;
     $$ LANGUAGE PLPGSQL;
 $ee$);
 
-CREATE OR REPLACE PROCEDURE SCHEMA_CATALOG.decompress_chunks_after(metric_table NAME, min_time TIMESTAMPTZ)
+CREATE OR REPLACE PROCEDURE SCHEMA_CATALOG.decompress_chunks_after(metric_table NAME, min_time TIMESTAMPTZ, transactional BOOLEAN = false)
```

Comment:
do we ever call this with `transactional => true`?

Reply:
Yes, we call it when turning compression off